CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This section contains changes that have been committed but not yet released.
 - Add support for filtering `metadata` using `metadata_key`, `metadata_value`, and `metadata_pair`]
 - Add support for new `RoomResource` fields
 - Add missing getters for Event.Recurrence fields
+- Add Neural API support
```

Comment:
I don't see any support for the Categorizer endpoint or the feedback requests yet.  Are those intended to be added later?  If so, perhaps we should be more specific in this changelog of what is supported so far.

Reply:
Feedback requests are not intended to be added later, but Categotizer is. I'll update the changelog to be more specific.